project/assets/asset_godot_scene.ts

```diff
@@ -326,20 +326,19 @@ export class AssetGodotScene extends BaseAsset {
       const className = rootSourceFile.exportedTsClassName()
 
       if (!className) {
-        addError({
-          description: `Failed to find classname for ${rootScript.fsPath}`,
-          error: ErrorName.Ts2GdError,
-          location: rootScript.fsPath,
-          stack: new Error().stack ?? "",
-        })
-
-        return "any"
+        // addError({
```

Comment:
Are there cases in which we still want to error here?

Reply:
It should be, but I think this could go as another PR because this one is already very big :)

Reply:
I still have some clean ups to do in the code. All commented out errors are not valid because their cases are handled properly and they should be removed.

Reply:
I still have some clean ups to do in the code. All commented out errors are not valid because their cases are handled properly and they should be removed.